.github/workflows/ci.yml

```diff
@@ -58,7 +58,44 @@ jobs:
           profile: Nexus 6
           script: |
             brew install parallel
-            parallel --retries 3 ::: "./gradlew connectedCheck"
+            parallel --retries 3 ::: "./gradlew sdk:connectedCheck"
+      - if: failure()
+        uses: actions/upload-artifact@v3
+        with:
+          name: androidTest-results
+          path: sdk/build/outputs/androidTest-results
+  benchmark-tests:
+    name: Android Benchmark
```

Comment:
This step take a long time. [See pipeline run](https://github.com/hCaptcha/hcaptcha-android-sdk/actions/runs/3461165663). Would it be possible to drastically reduce the time?

![image](https://user-images.githubusercontent.com/11096226/201656365-8d6a866b-c11c-4a0e-a60a-e7e5603f11b3.png)

Reply:
I'm not sure but I can research this topic. We can try some dirty hacks like https://stackoverflow.com/a/69318395/902217 but this kills the whole idea of benchmarking.

I think we should leave `'androidx.benchmark'` plugin alone and allow it to decide how many runs it needs

Also, we can improve this "problem" from another side by buying a more performant GH runner (on my MBP `./gradlew bench:connectedReleaseAndroidTest` takes 2m 7s